styles.css

```diff
@@ -71,14 +71,247 @@
   margin-bottom: 12px;
 }
 
-ul {
+ul.social-media {
   list-style-type: none;
   margin-bottom: 120px;
   margin-left: 24px;
   margin-right: 24px;
 }
 
-ul li {
+ul.social-media li {
   display: inline;
   margin: 10px 12px;
 }
+
+#works-section-title {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 52px;
+  color: #1c1a19;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 24px;
+  text-align: center;
+}
+
+#separator {
+  border-bottom: 1px solid #212121;
+  width: 327px;
+  margin: auto;
+  margin-bottom: 80px;
+}
+
+#featured-project {
+  margin-right: 24px;
+  margin-left: 24px;
+}
+
+#featured-project-image {
+  display: block;
+  margin: auto;
+  margin-bottom: 24px;
+}
+
+#featured-project-title {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+#featured-project-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 12px;
+}
+
+ul.featured-project-languages {
+  list-style-type: none;
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  gap: 12px;
+}
+
+ul.featured-project-languages li {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  text-align: center;
+  padding: 12px 10px;
+}
+
+.featured-project-button {
+  align-items: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  color: #fff;
+  border: none;
+  margin-top: 12px;
+}
```

Comment:
- [ ] Nice job for implementing this button section to march the design. You can improve the user experience by adding more button styling: _Active_, _Hover_, and _Pressed_ ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27))